api/src/messaging/api_views.py

```diff
@@ -33,6 +43,41 @@ def get_queryset(self):
         return self.request.user.rooms.all()
 
 
+class MembershipCreateDestroy(APIView):
```

Comment:
Maybe change this name to something that describes what this API view does, which is to add or remove a user from a room